src/script/util/LoggerUtil.js

```diff
@@ -29,7 +29,8 @@ function enableLogging(force = false, search = window.location.search) {
 
   try {
     localStorage = window.localStorage;
-  } catch (error) {
+  } catch (error) {}
+  if (!localStorage) {
```

Comment:
Let's add your comments ("This could be due to a Firefox regression...") as a code comment to not forget why we check for `!localStorage`.

Reply:
I moved the comment a few lines up here and added the note. Does it make sense?

Reply:
Yes, thanks for adding the details.